scala/spark-wordcount/pom.xml

```diff
@@ -9,11 +11,12 @@
     <version>0.1</version>
 
     <properties>
-        <bigtable.version>1.7.0</bigtable.version>
-        <hbase.version>1.3.1</hbase.version>
+        <bigtable.version>1.15.0</bigtable.version>
+        <hbase.version>1.5.0</hbase.version>
```

Comment:
Please add a comment that the hbase & hadoop versions follow dataproc so that in the future we can stay consistent

Reply:
Also mention what the current version of dataproc we are tracking

Reply:
The first comment addressed. How to find out the version of Dataproc?

Reply:
Done